packages/beacon-node/src/sync/range/chain.ts

```diff
@@ -439,9 +438,10 @@ export class SyncChain {
 
       // At least one block was successfully verified and imported, so we can be sure all
       // previous batches are valid and we only need to download the current failed batch.
-      if (res.err instanceof ChainSegmentError && res.err.importedBlocks > 0) {
-        this.advanceChain(batch.startEpoch);
-      }
+      // TODO: Disabled for now
```

Comment:
can you make an issue for this? I guess it's a small thing, an optimization to avoid reprocessing up to batch.length - 1 blocks.

Reply:
We can do an issue but I don't see this ever being implemented judging for our priorities in the last 2 years